js/script.js

```diff
@@ -0,0 +1,35 @@
+﻿var tweetLink = "https://twitter.com/intent/tweet?text=";
+var quoteUrl = "https://quotesondesign.com/wp-json/posts?filter[orderby]=rand&filter[posts_per_page]=1";
+var prefix = "https://cors-anywhere.herokuapp.com/";
+
+function getQuote() {
+	$.ajaxSetup({ cache: false });
+	$.getJSON(prefix + quoteUrl, createTweet);
+}
+
+function createTweet(input) {
+	var data = input[0];
+	var quoteText = $(data.content).text().trim();
```

Comment:
`var quoteText = data && data.content ? $(data.content).text().trim() : null` - what will this line do?